examples/hello-world/tslint.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "$schema": "http://json.schemastore.org/tslint",
-  "extends": ["./node_modules/@loopback/build/config/tslint.build.json"]
+  "extends": ["./node_modules/@loopback/tslint-config/tslint.build.json"]
```

Comment:
We should use the module name instead:
```
"extends": ["@loopback/tslint-config/tslint.build.json"]
```
See https://palantir.github.io/tslint/usage/configuration/

Reply:
Ah, of course, good catch!

Reply:
Fixed.